### ISSUE TITLE
Questions in this repository

### ISSUE THREAD

**Issue by brianebeling** [corona-warn-app/cwa-website#3286]
### Discussed in https://github.com/corona-warn-app/cwa-website/discussions/3280

<div type='discussions-op-text'>

<sup>Originally posted by **MikeMcC399** December  7, 2022</sup>
Now that [discussions](https://github.com/corona-warn-app/cwa-website/discussions) have been enabled in this repository, should all questions now be placed here instead of in [issues](https://github.com/corona-warn-app/cwa-website/issues)?

If yes, then existing open [questions](https://github.com/corona-warn-app/cwa-website/labels/question) should be moved to [discussions](https://github.com/corona-warn-app/cwa-website/discussions).

Also the templates in https://github.com/corona-warn-app/cwa-website/tree/master/.github/ISSUE_TEMPLATE would need to be modified to point questions to [discussions](https://github.com/corona-warn-app/cwa-website/discussions).</div>

**Comment by brianebeling** [comment]
I accidentally closed it.
The Issue_Template still needs to be updated. The other tasks were already completed.

**Comment by Ein-Tim** [comment]
I could provide a PR for this, if you don't want to do it yourself?

**Comment by brianebeling** [comment]
> I could provide a PR for this, if you don't want to do it yourself?

Sure, that would be great! I'm currently working on other things.

**Comment by MikeMcC399** [comment]
@Ein-Tim 

How do you propose to solve this?

One way would be to 

- delete https://github.com/corona-warn-app/cwa-website/blob/master/.github/ISSUE_TEMPLATE/02_questions.md and 
- add a link to [discussions ](https://github.com/corona-warn-app/cwa-website/discussions) into https://github.com/corona-warn-app/cwa-website/blob/master/.github/ISSUE_TEMPLATE/01_site_issue.md

**Comment by Ein-Tim** [comment]
@MikeMcC399 

I would start with adding a link to the issue template and after some grace period, in which all contributors are informed about the fact that questions should now be asked via the discussions feature, we could completely remove the question template. 

What do you think?

**Comment by MikeMcC399** [comment]
@Ein-Tim 

That would be a cautious way of implementing it. I don't think you need to be so careful if you look at who has been submitting issues lately! I suggest to wait for feedback from @brianebeling whether to do one step or two steps.

**Comment by Ein-Tim** [comment]
@MikeMcC399 

Yes, you're probably right, the most active community contributors (you & me 🙂) will know about this change.

As you suggested, I'll wait for feedback from @brianebeling!

**Comment by brianebeling** [comment]
I mean @MikeMcC399 is not wrong. The number of people still actively contributing is not a lot currently. More so you two and the other maintainers. At least in this repo. I'd still go with the cautious approach, because I'm not sure how the change is perceived, yet. Leaving it there for a while doesn't do us any harm and if someone uses the template that translates to a nice initial post that is structured for a discussion when transferred, at least.

**Comment by Ein-Tim** [comment]
Okay, PR will follow tomorrow!

**Comment by MikeMcC399** [comment]
@Ein-Tim 

Are you still planning to provide a PR for this?

**Comment by Ein-Tim** [comment]
@MikeMcC399, for sure! Sorry, it slipped through. PR will follow this afternoon.

**Comment by Ein-Tim** [comment]
PR https://github.com/corona-warn-app/cwa-website/pull/3291 provided.

**Comment by brianebeling** [comment]
We talked internally about the discussion feature. It hasn't had a strong negative impact, but its also hard to point out a strong positive. Personally speaking I think it was a good experience for possible future projects, but its hard to say how good it scales up to very active repositories and there have been some pain points that we likely cannot resolve in due time. So a decision has been made to go back to Issues only for this repository, to simplify the development experience and not introduce an all new approach to organization at this point in time. 

Thank you for your understanding and participation. I will later convert things back to how they were before. How did you perceive the change?

Cc: @Ein-Tim @MikeMcC399 @dsarkar

**Comment by Ein-Tim** [comment]
> I will later convert things back to how they were before. 

That sounds good! 

> How did you perceive the change?

I think that discussions are a useful tool, especially for informational only *issues*. However, before I would deploy the discussions feature to all repositories of a GitHub organization the size of this one with very active development and community (like the CWA repositories at the beginning of the project), I would certainly make more tests, as I fear two things:
- information is spread out over too many places and gets lost
- is maintaining the discussions feature AND the issues feature worth it, or does it, in the end, come down to having two features for one and the same thing

Thank you for sharing your & the team's experience on the feature!

**Comment by MikeMcC399** [comment]
@brianebeling 

Would it be possible to leave Discussions in place for this repository only for the remaining life-time of the project? I found them to be a useful extension for topics which did not need to be Issues.

What were your pain-points in handling discussions here? Did we raise too many topics? I can only see @Ein-Tim and myself as actually opening Discussions.

If Discussions are closed here, then I don't expect to share information in future that needs no action, since this doesn't fit the model for Issues.

I was never a fan of enabling Discussions in other repositories as I tried to make clear in https://github.com/corona-warn-app/cwa-website/discussions/3282, however I thought they were useful here in the cwa-website repository.
